src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -79,6 +86,22 @@ public CaseDetails saveCountyCourtJudgment(
         return this.update(authorisation, ccdCase, DEFAULT_CCJ_REQUESTED);
     }
 
+    public CaseDetails saveDefendantResponse(
```

Comment:
Can you please make argument list consistent with other places. We should have the following order.

```
String defendantId,
String defendantEmail,
Response response,
```